operator/main.go

```diff
@@ -100,7 +103,13 @@ func run() error {
 	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
 
-	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
+	zapLogger := zap.NewRaw(zap.UseFlagOptions(&opts))
```

Comment:
Not sure, but maybe the flag.Parse need to be after `zap.UseFlagOptions` ? 
```
opts := zap.Options{}
opts.BindFlags(flag.CommandLine)
flag.Parse()
log := zap.New(zap.UseFlagOptions(&opts))
```

see https://github.com/kubernetes-sigs/controller-runtime/blob/v0.15.0/pkg/log/zap/zap.go#L307

Reply:
I'm not sure I follow. In your example `flag.Parse` is **before** `zap.UseFlagOptions` just like in the source code.
Is there anything I'm missing?

Reply:
hm, my bad, I think my brain had a hiccup when I read this the first time. Looks good!